torch/quantization/convert_modules.py

```diff
@@ -0,0 +1,222 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch.nn as nn
+import torch.nn.quantized as nnq
+import torch
+from .observer import *
+import heapq
+
+def _forward_hook(self, input, output):
+    for module in self.children():
+        module.forward(output)
+
+def add_observer(module, qconfig_dict, qconfig_parent=None, prefix=''):
```

Comment:
it's probably useful to expose this method as part of public API too. If so - keep the add_observer with just two args and have a recursive function to be a helper function (so that we don't expose prefix and qconfig_parent as part of docs)

Reply:
still not exposed as part of public api? let's do that

Reply:
I agree, so this method just adds an observer to a single module, correct?

Reply:
I agree, so this method just adds an observer to a single module, correct?